proto/cosmos/tx/signing/v1beta1/signing.proto

```diff
@@ -21,6 +21,10 @@ enum SignMode {
   // from SIGN_MODE_DIRECT
   SIGN_MODE_TEXTUAL = 2;
 
+  SIGN_MODE_DIRECT_AUX = 3;
+
+  SIGN_MODE_TEXTUAL_AUX = 4;
```

Comment:
Assuming we have `SIGN_MODE_DIRECT` and `SIGN_MODE_DIRECT_JSON` (as discussed in https://github.com/cosmos/cosmos-sdk/pull/8476#discussion_r580192281), should we have?

```suggestion
  SIGN_MODE_AUX = 3;

  SIGN_MODE_AUX_JSON = 4;
```

(note: I actually don't understand what "direct" and "aux" stand for, so i'm maybe missing something obvious).